xLights/sequencer/tabSequencer.cpp

```diff
@@ -89,15 +90,21 @@ void xLightsFrame::CreateSequencer()
     logger_base.debug("                Set dock size constraints.");
     m_mgr->SetDockSizeConstraint(0.25, 0.15);
 
+    const auto maxHeight = std::max(1, wxDisplay().GetGeometry().GetHeight() - 100);
+    const auto maxWidth = std::max(1, wxDisplay().GetGeometry().GetWidth());
```

Comment:
How would this work for those of us that are using multiple displays, possibly/likely at different resolutions?    I'd honestly prefer NOT setting any sort of MaxHeight on anything so that the windows can easily be moved to different displays and still be maximized.

Reply:
If it goes off the screen you cant get to the bottom. The double click still maximizes it. The original code was setting the max off your screen. GetHeight should return the largest your monitor can handle.  Not sure how it handles resolutions and such though. This code is similar to code used in Audacity.

Reply:
But that's part of the problem....   My "main" monitor has a height of 1620 (logical size), but if the window appears on notebook screen, it has a max of 1329 and it gets cut off.

Reply:
The current code is hard coded to 1050. This is better than hard coded arbitrary value. Is there an even better way? If you dont have it set, you end up having to resize the box every time.